mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -127,10 +128,16 @@ final class OneVsRestModel private[ml] (
       predictions.maxBy(_._2)._1.toDouble
     }
 
+    // output the index of the classifier with highest confidence as prediction
+    val probabilityUDF = udf { (predictions: Map[Int, Double]) =>
```

Comment:
This is outputting a Double, but it should return a Vector with one value for each class label.  Actually, the output value should be the accCol converted to a Vector.